routers/web/user/setting/profile.go

```diff
@@ -137,11 +137,8 @@ func ProfilePost(ctx *context.Context) {
 		return
 	}
 
-	// Update the language to the one we just set
```

Comment:
Maybe we need to detect whethere ctx.Doer.Language changed here?

Reply:
I do not think so.

Before, the language setting was on the profile page, so the code was needed.

But right now, the language setting is on the appearance page, so the profile page shouldn't touch the language anymore.

* #17433